src/unity/python/turicreate/toolkits/classifier/logistic_classifier.py

```diff
@@ -36,6 +36,7 @@ def create(dataset, target, features=None,
     max_iterations = _DEFAULT_SOLVER_OPTIONS['max_iterations'],
     class_weights = None,
     validation_set = 'auto',
+    seed=None,
```

Comment:
Same comment here: can we preserve API compatibility?

Reply:
Done.